app/src/main/java/com/example/sabonit/UserLoginActivity.java

```diff
@@ -93,11 +91,6 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                     // FirebaseUser.getIdToken() instead.
                     uid = user.getUid();
                     updateAccount();
-//                    if (! isAccountExist)
```

Comment:
why did you remove it? 
did you check that everything works properly?

Reply:
i think so, checked it on 2 devices and need to notice if it override new account, in my checks - it didnt